bin/admin/twistedchecker-trunk-diff.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+#
+# Helper for running twistedchecker and reporting only errors that are part
```

Comment:
If we are going to have helpers that developers are expected to run, they should be in Python, not shell scripts.  Shell should only be completely travis-specific stuff.  Shell scripts, among other problems, don't run on Windows.